sql_joins/sql_joins.md

```diff
@@ -0,0 +1,875 @@
+
+**Learning Objectives**
+
+@learning_objectives
+
+</div>
+
+## Multiple Tables
+
+Most SQL queries require something more complex than referencing data from a single table. This is where SQL join functionality and the `JOIN` command come into play.
+
+SQL joins are used to combine rows from two tables, based on some set of columns they have in common.  For example, consider the case where you have data about a multi-site study's research subjects. One table holds depression scores for subjects and a different table holds subject addresses.  Your hypothesis is that people who live in certain zip codes have higher rates of depression.  
+
+To see if your hypothesis has evidence to back it, you need to combine data, taking the subject ID and depression score from one table, and the subject ID and zip code from another table, and combining them, so that you get matching information in the same table row.
+
+Maybe your source tables look something like the tables below:
+
+**Table 1: depression\_scale**
+
+<!-- data-type="none" class="tight-table" style="font-size:80%"-->
+| subj_id  | date  | dep_q1  | dep_q2   | dep_q3  | dep_q4  | dep_total   |
+| :--------- | :--------- | :--------- | :--------- | :--------- | :--------- | :--------- |
+| 11234   | 2021-05-15   | 3    | 3  | 2    | 4    | 12    |
+| 86234   | 2021-06-01   | 4    | 4  | 3    | 4    | 15    |
+| 32660   | 2021-06-10   | 1    | 1  | 2    | 1    | 5   |
+| 86234   | 2022-01-13   | 2    | 2  | 1    | 3    | 8    |
+| 41356   | 2022-02-10   | 1    | 3  | 2    | 3    | 10   |
+
+**Table 2: subject\_address**
+
+<!-- data-type="none" class="tight-table" style="font-size:80%"-->
+| subj_id  | street_address  | city  | state   | zip  | date_start  | date_end   |
+| :--------- | :--------- | :--------- | :--------- | :--------- | :--------- | :--------- |
+| 11234   | 123 Main Street   | Smithtown    | PA  | 19000    | 2022-01-01   | `NULL`    |
+| 11234   | 123 Oak Lane   | Old Towne    | PA  | 18000   | 2000-01-01    | 2021-12-31    |
+| 93452   | 123 Green Blvd  | Kirby    | TN  | 37000    | 2020-05-01    | `NULL`   |
+
+What you want to eventually end up with will be a single table of results that might only have three columns: "subj\_id", "dep\_total", and "zip".  That way you can look at the relationship between depression inventory scale and zip code.
+
+## Forming a JOIN 
+
+There are two basic pieces of information you need to know to write successful joins:
+
```

Comment:
Would it be reasonable to put an entire statement SELECT... FROM... JOIN... ON.. here as an example? It might be helpful to have one "complete" example of how this all fits in before putting ellipses at the start of the code block for the rest of the module, just to remind people what the ellipses are eliding.

Reply:
agree! changed as the third, fullest example